go.mod

```diff
@@ -292,3 +292,5 @@ exclude (
 	k8s.io/client-go v9.0.0+incompatible
 	k8s.io/client-go v9.0.0-invalid+incompatible
 )
+
+replace github.com/kube-object-storage/lib-bucket-provisioner => github.com/BlaineEXE/lib-bucket-provisioner v0.0.0-20221102211652-aa9aebb5b74f
```

Comment:
planned for revert I assume

Reply:
Yes. We need to merge the L-B-P changes before this PR.

Reply:
That's done now. If things are looking good, I'll merge once it seems like CI is stable.

Reply:
Note to self: I also want to go through the upgrade test after my latest changes. It went well a few days ago, but I should double check.